twill-core/src/main/java/org/apache/twill/internal/TwillRuntimeSpecification.java

```diff
@@ -72,7 +76,7 @@ public String getZkConnectStr() {
     return zkConnectStr;
   }
 
-  public RunId getTwillRunId() {
+  public RunId getTwillAppRunId() {
```

Comment:
I changed it because in {code}TwillContainerMain{code}, the main method will try to get the run id for both the application and the run id for the TwillRunnable. And since TwillRuntimeSpecification is mainly to pass runtime information for TwillApplication, I feel like it is clearer to name like this.